demo/asset_section.js

```diff
@@ -57,6 +58,14 @@ shakaDemo.setupAssets_ = function() {
     if (asset.drm.length && !asset.drm.some(
         function(keySystem) { return shakaDemo.support_.drm[keySystem]; })) {
       option.disabled = true;
+      asset.drm.forEach(function(keySystem) {
```

Comment:
Joey's suggestion was to use the drm probe to detect browser support.  `asset.drm` is the key systems available in the asset.  You can use `shakaDemo.support_.drm[keySystem]` to detect if it is supported on the browser.

Reply:
Correct.  We are less interested in what key systems the selected asset uses.  If the "custom" field is used, we don't know.  But if the browser supports Widevine, it is reasonable to suggest Widevine-specific values here regardless of what asset is selected.